### ISSUE TITLE
If it will have a great impact on algorithm performance?

### ISSUE THREAD

**Issue by liwensheng2020** [zhanghang1989/ResNeSt#138]
Hello,
I integrated ResNeSt into UNet to extract feature maps, and I set batch size to 1. At this time, an error occurred at gap=self.bn1(gap) in splat.py. I comment out this line and it worked, but I don't know if it will have a great impact on algorithm performance.

**Comment by liwensheng2020** [comment]
I'm trying to train a model, and limited to the GPU memory, I have to set the batch size to 1. Which of the following two operations will have less impact on performance: 1) Decrease model depth[I used resnest50 right now]; 2) Comment out that line.

**Comment by zhanghang1989** [comment]
If you only train with batch size = 1, I would suggest using `model.eval()` instead of `model.train()` and reduce the learning rate.

**Comment by liwensheng2020** [comment]
Thank you
